src/main/scala/loamstream/drm/DrmJobWrapper.scala

```diff
@@ -57,7 +57,7 @@ final case class DrmJobWrapper(
 
     val result = s"${singularityPart}${commandLineJob.commandLineString}"
 
-    debug(s"Raw command in DRM shell script: '${result}'")
+    trace(s"Raw command in DRM shell script: '${result}'")
```

Comment:
Just curious what the difference here is? I assume a log level, but want to make sure (is this a pattern I can adopt?).

Reply:
Trace is just a lower level than debug, like debug is a lower level than info, or warn.